tests/ci/run_ec2_test_framework.sh

```diff
@@ -77,6 +76,9 @@ for i in {1..30}; do
   sleep 60
 done
 
+# Wait 5 minutes for instance to "warm up"?
+echo "Instances need to initialize a few minutes before SSM commands can be properly run"
+sleep 300
```

Comment:
Instead of just waiting can this check some SSM agent status and ensure it's up and running before starting?